### ISSUE TITLE
Routing/ Price incrementation

### ISSUE THREAD

**Issue by iDome89** [iDome89/NewReactProject#1]
I am stuck with this "fake app", don't look at the styling since it is complete garbage, it is just for exercising.

I'd like to  make a clone of howmuchtomakeanapp but I can't find I way to make the routing work and I would like to improve the logic for the price incrementation to make it possible with a back button to go to the previous choice and change the price accordingly.

**Comment by zerho** [comment]
who said that it's fake? if it works it works @iDome89

**Comment by iDome89** [comment]
@zerho <3

**Comment by Max93** [comment]
Hi man (@iDome89), yes sure I can help u. I will not look the styling (pinky promise). For the first u need a state container like REDUX. Do you know what we're talking about?

**Comment by iDome89** [comment]
@Max93 yes I know what redux is but I haven’t used it yet and I didn’t start learning it. I am just at the beginning of my React journey. I am still stuck at the routing which I am unable to make work in the app. What I mean is, click on the image -> go to another “page”.  I would be glad if you can mentor me a little bit in this. Maybe @zerho can give you my contact if you want.
